### PR TITLE
Move to 100% Haml views

### PR DESCRIPTION
**Pull request by sardaukar**
Getting rid of the few ERB views we have.

### CODE REVIEW

**Comment by teoljungberg** [comment]
I'm personally in the ERB camp, but I know the rest of the team prefer HAML.

**Comment by sardaukar** [comment]
@teoljungberg same, but having both is kind of odd. And Haml had a recent release, so it doesn't seem to be dead anymore.

File: app/views/shared/_plan_vertical.html.haml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+%div{ :class => "plan plan-vertical #{current ? 'current' : ''}" }
```
Comment (croaky) [inline app/views/shared/_plan_vertical.html.haml]: `class:` ?

Comment (sardaukar) [inline app/views/shared/_plan_vertical.html.haml]: Argh, I used `erb2haml` and this showed up. I need to read up on Haml to do this the 'proper' way.

**Pull request merged**